ios/FluentUI/Navigation/UIBarButtonItem+BadgeValue.swift

```diff
@@ -24,13 +25,21 @@ import UIKit
         }
     }
 
-    /// Convenience method to set the badge value to a number.
-    /// If the number is zero, the badge value will be hidden.
-    @objc func setBadgeNumber(_ number: UInt) {
-        if number > 0 {
-            badgeValue = NumberFormatter.localizedString(from: NSNumber(value: number), number: .none)
-        } else {
-            badgeValue = nil
+    @objc internal var badgeAccessibilityLabel: String? {
+        get {
+            return objc_getAssociatedObject(self, &AssociatedKeys.badgeAccessibilityLabel) as? String ?? nil
+        }
+        set {
+            objc_setAssociatedObject(self, &AssociatedKeys.badgeAccessibilityLabel, newValue, .OBJC_ASSOCIATION_RETAIN_NONATOMIC)
         }
     }
+
+    /// Use this method on bar button item's instance to set the badge value and badge accessibility label.
+    /// Set the badge value to nil to hide the red oval.
```

Comment:
nit: using the documentation standard for each parameter might be clearer to inform about values that can be set and their consequences in the rendering of the control.

You can hit `command + option + /` to have Xcode generate the template automatically.

Reply:
Done!!